presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveSplitSource.java

```diff
@@ -286,17 +315,22 @@ private TestSplit(int id)
         }
 
         private TestSplit(int id, OptionalInt bucketNumber)
+        {
+            this(id, bucketNumber, DataSize.ofBytes(100));
+        }
+
+        private TestSplit(int id, OptionalInt bucketNumber, DataSize fileSize)
```

Comment:
I guess you can avoid adding an overload, if you define 100 bytes as the const
and use that in test code. This way test code is slightly more complicated (bad),
but common code remains simpler.